librz/include/rz_bin.h

```diff
@@ -739,6 +761,10 @@ RZ_IPI void rz_bin_section_free(RzBinSection *bs);
 RZ_API void rz_bin_info_free(RzBinInfo *rb);
 RZ_API void rz_bin_import_free(RzBinImport *imp);
 RZ_API void rz_bin_symbol_free(RzBinSymbol *sym);
+static inline bool rz_bin_reloc_has_target(RzBinReloc *reloc) {
+	return reloc->target_vaddr && reloc->target_vaddr != UT64_MAX;
```

Comment:
Do you mean calling it `rz_bin_reloc_has_target_vaddr` instead of `rz_bin_reloc_has_target`? Or what exactly do you mean?

Reply:
The latter. But this is really more of a general thing... In many places we check if addr != 0 && addr != UT64_MAX, but by doing so we implicitly assume that 0 and UT64_MAX are never valid values... I wonder if this assumption is always true.

Reply:
I am quite sure it is not always true. I think we might want to think about a ghidra-like scheme where an address is always space + 64bit offset. Then the space variable could be directly re-used to tell whether it's an invalid address instead of adding another bool.

Reply:
Yeah probably as first thing we should refactor all ut64 used as addr to a typedef or struct `RzAddr` that can be implemented as necessary.. Anyway, challenge for another day :D